src/module-3/online-store/styles/base/_card.scss

```
@@ -0,0 +1,115 @@
+@use 'global' as *;
+
+.card {
+  height: 486rem;
+  //max-width: 400rem;
+  width: 100%;
+  @extend %def-box;
+  background: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  &__product {
+    margin-top: 20rem;
+    height: 272rem;
+    width: 238rem;
+    border-radius: 0;
+    position: relative;
+  }
+
+  .info {
+    width: 90%;
+    display: inherit;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    &__rate {
+      height: 30rem;
+      width: 65rem;
+      border-radius: 3rem;
+      display: inherit;
```

```suggestion
      display: flex;
```   otherwise you'll lose your flex styles if parent is not flex anymore